doc/user/content/ops/diagnosing-using-sql.md

```diff
@@ -136,6 +136,13 @@ group by mdo.id, mdo.name
 order by sum(mas.records) desc;
 ```
 
+We've also bundled a [memory usage visualization tool](https://materialize.com/docs/ops/monitoring/#memory-usage-visualization)
+to aid in debugging. The sql queries above show all arrangements in Materialize
+(including system arrangements), whereas the memory visualization tool shows
+only user-created arrangements, and it shows them by dataflow. The amount of
+memory used by Materialize should correlate with the number of arrangement
+records that are displayed by the visual interface.
```

Comment:
Technically the memory used by Materialize correlates with the number of arrangement records displayed by either the queries or the visual interface.

Reply:
good point!  just updated